### PR TITLE
chapter19_part1: /210_Identifying_Words/00_intro.asciidoc 

### PR DESCRIPTION
**Pull request by lxy4java**
Thanks for the Pull Request!  We really appreciate your help and contribution!

Before you submit the PR, have you signed Contributor License Agreement: https://www.elastic.co/contributor-agreement/ ?

PR's (no matter how small) cannot be merged until the CLA has been signed.  
It only needs to be signed once, however. Thanks!

### CODE REVIEW

File: 210_Identifying_words/00_Intro.asciidoc [hunk 1]
```diff
@@ -1,28 +1,24 @@
...
+然而即使在英语词汇中也会有一些争议： _you're_ 是一个单词还是两个？ _o'clock_ ， _cooperate_ ， _half-baked_ ，或者 _eyewitness_ 这些呢？
+
+
+
+德语或者荷兰语把独立的单词合并起来创造一个长的合成词如 _Weißkopfseeadler_ (white-headed sea eagle) ,
+但是为了在查询 `Adler` (eagle)的时候返回查询 `Weißkopfseeadler` 的结果，我们需要懂得怎么将合并词拆成词组。
+
+
+
+亚洲的语言们更复杂：很多语言在单词，句子，甚至段落之间没有空格。((("Asian languages", "identifying words")))
```
Comment (medcl) [inline 210_Identifying_words/00_Intro.asciidoc]: 『们』有点多余

File: 210_Identifying_words/00_Intro.asciidoc [hunk 2]
```diff
@@ -1,28 +1,24 @@
...
+有些词可以用一个字来表达，但是同样的字在另一个字旁边的时候就是不同意思的长词的一部分。
+
+
+
+显而易见的是没有能够奇迹般处理所有人类语言的万能分析器，Elastticsearch 为很多语言提供了专用的分析器，
+其他特殊语言的分析器以插件的形式提供。
+
+
+然而并不是所有语言都有专用分析器，而且有时候你甚至无法确定处理的是什么语言。这种情况，我们需要一些忽略语言后能合理工作的标准工具包。
```
Comment (medcl) [inline 210_Identifying_words/00_Intro.asciidoc]: 忽略语言后能合理
=》
忽略语言也能合理
？

Comment (richardwei2008) [inline 210_Identifying_words/00_Intro.asciidoc]: Elastticsearch -> Elasticsearch

Comment (lxy4java) [inline 210_Identifying_words/00_Intro.asciidoc]: @medcl  @richardwei2008  谢谢。

**Comment by gitqh** [comment]
LGTM @lxy4java

**Comment by node** [comment]
LGTM

**Pull request merged**